packages/apputils/style/dialog.css

```diff
@@ -121,11 +121,6 @@ button.jp-Dialog-close-button {
   padding-right: 5px;
 }
 
-.jp-Dialog-checkbox > input:focus-visible {
-  outline: 1px solid var(--jp-input-active-border-color);
-  outline-offset: 1px;
-}
-
```

Comment:
(Specifically for checkboxes in dialogs)
for context, this styling is applied by `packages/apputils/src/dialog.tsx` by using the label class as its initial selector.

<img width="736" alt="image" src="https://github.com/jupyterlab/jupyterlab/assets/113391940/44efb32b-5021-474d-83f3-e1872f0a5dfa">

But further down the execution create footer calls `Styling.style(footer.node);` which adds `jp-mod-styled` to the checkbox element. Deleting this styling in`packages/apputils/style/dialog.css` reduces code duplication.
<img width="492" alt="image" src="https://github.com/jupyterlab/jupyterlab/assets/113391940/8074c649-53e9-4875-a317-451289217c46">